Probesys/bileto
# New/show authorized organizations on home page

## Related issue(s)



https://github.com/Probesys/bileto/issues/207

## Changes



- Improve spacing of cards so they look better
- Load accessible organizations on the home page and display them similarly to the organizations page

## How to test manually



- give authorization to one specific organization to your user → check it is visible on the home page
- give global authorization to your user → check all the organizations are visible on the home page
 
## Checklist
